pythonforandroid/recipes/m2crypto/__init__.py

```diff
@@ -1,47 +1,41 @@
-from pythonforandroid.recipe import PythonRecipe
-from pythonforandroid.toolchain import current_directory, shprint
-from os.path import join
+from pythonforandroid.recipe import CompiledComponentsPythonRecipe
+from pythonforandroid.toolchain import current_directory
+from pythonforandroid.logger import shprint, info
+import glob
 import sh
 
-
```

Comment:
Thanks for updating that pull request.
Linter is complaining because you removed that line :stuck_out_tongue: 
https://travis-ci.org/kivy/python-for-android/jobs/398189288
You can run `tox` on your dev env so you don't have to wait for Travis to fail to fix the linter.